tests/ios/swift-version/swift_version_tests.bzl

```diff
@@ -0,0 +1,45 @@
+load("//rules:framework.bzl", "apple_framework")
+load("@build_bazel_rules_swift//test/rules:action_command_line_test.bzl", "action_command_line_test")
+
+def apple_framework_swift_version_test(*, name, srcs = [":empty.swift"], given, expected, not_expected = []):
+    framework_name = name + "_" + given.replace(".", "_")
```

Comment:
Any reason we _need_ to replace `.` with `_` here?

Reply:
I dont think so, but IMO the underscores make for nicer target names